### ISSUE TITLE
Preview LC0075 - accept smaller codes in .Get() calls

### ISSUE THREAD

**Issue by tkroes** [StefanMaron/BusinessCentral.LinterCop#858]
Take for example the following table:
```al
table 50000 "Test Table"
{
    fields
    {
        field(1; PK; Code[20])
        {
            Caption = 'PK';
            NotBlank = false;
        }
    }
}
```
If `.Get()` is called on this table with a `Code[20]`, there is no issue. Calling it with a code of any other length raises an error. However it should be acceptable to use a `.Get()` with a smaller code, as this does not cause a runtime error, whereas a variable with a greater length could.
```al
procedure TestFunction(SmallCode: Code[10]; CorrectCode: Code[20]; BigCode: Code[50])
var
    TestTable: Record "Test Table";
begin
    TestTable.Get(CorrectCode); //No error raised, correct usage
    TestTable.Get(BigCode);     //LC0075 raised, correct as variable is too big for PK
    TestTable.Get(SmallCode);   //LC0075 gets raised incorrectly, as variable fits within PK
end;
```

**Comment by Arthurvdv** [comment]
You're right, the rule shouldn't be raise on the SmallCode variable in your example.

This false positive is now fixed in the updated (pre)release version [v0.31.4](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.31.4) of the LinterCop.
